user_guide/intro_navigation.rst

```diff
@@ -0,0 +1,110 @@
+
+|
+
+In the mobile version the menu bar can be accessed via the "hamburger" button next to the organization name:
+
+|
+    
+.. image:: ./img/navigation/menu/header_mobile.png
+
+|
+
+The list of menu items available to you, depends on the access and permission settings defined in the organization.
+
+Hover the mouse over a menu item to expand the drop-down menu with the second-level items. Click on them to access 
+corresponding pages or grids. (See :ref:`Data management basics <user-guide-data-management-basics>` for further 
+details.)
+  
+  
+.. _user-guide-navigation-panel:
+
+Navigation Tools
+----------------
+
+At the very top of the page, there is navigation panel that contains different navigation tools, as shown below.
+
+|
+
+.. image:: ./img/navigation/panel.png
+
+
+.. csv-table::
+  :header: "#","Item","Description","Related Document"
+  :widths: 10, 30, 40,30
+  
+  "1","**Organization Selector**","Switch between organizations when working in  the multi-organization environment.","
+  :ref:`Working with Multiple Organizations <user-guide-getting-started-change-organization>` section of the 
+  *Company Structure and Organization Selector* guide`"
+  "2","**Shortcuts**",Quickly perform the most common actions from anywhere in the system.","
+  :ref:`Shortcuts <user-guide-getting-started-shortcuts>`"
+  "3","**Search**","Find a specific record's 
+  :ref:`View page <user-guide-ui-components-view-pages>`","
+  :ref:`Search <user-guide-getting-started-search>`"
+  "4","**Pinned pages**","Open pages preserved in the state they were pinned in.",
+  ":ref:`Pinned Page <user-guide-ui-components-pinned>`"
+  "5","**User menu**","Access your user profile, calendar, mailbox, and task list. Log out of the system.","
+  :ref:`User Menu <user-guide-getting-started-user-menu>`"
+  "6","**My Emails**","See if you have new and unread emails. Quickly read and reply to most recent emails. 
+  
+  The envelop icon reflect current state of your mailbox - from 
+  the menu under it, you can get to the *My Emails* page, read and reply any of the most recent 5 emails and mark all 
+  the emails as read",""
+  "7","**Hamburger menu**","Access your
+
+  * recently opened pages
+  * favorite pages
+  * most viewed pages","
+  :ref:`History, Favorites and Most Viewed <user-guide-getting-started-history>`"
+  "7","**Help Icon**","Access the User Guide",""
+  "8","**Sidebar panel**","Hosts widgets that provide fast and convenient access to the constantly needed information, 
+  such as recent emails or the task list. ",":ref:`Sidebar Pannel <user-guide-navigation-sidebar-panel>`"
+  
+
+This way, the users can easily access, store and organize any and all the information required to manage the customer 
+relations in the most efficient way.
+
+Flash notifications and Dialog windows
+--------------------------------------
+
+Flash notifications and dialog windows are used by OroCRM when it notifies you about something important.
```

Comment:
Edit: Flash notifications and dialog windows are used by OroCRM to notify you about something important.